backend/src/availability/availability.module.ts

```diff
@@ -1,10 +1,11 @@
 import { forwardRef, Module } from '@nestjs/common';
 import { TypeOrmModule } from '@nestjs/typeorm';
-import { LessonsModule } from 'lessons/lessons.module';
-import { InstructorsModule } from 'instructors/instructors.module';
 import { AvailabilityService } from './availability.service';
 import { AvailabilityController } from './availability.controller';
 import { Availability } from './entities/availability.entity';
+import { InstructorsModule } from '../instructors/instructors.module';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Those `eslint-disable-next-line import/no-cycle` directives are expected here, as we actually have circular dependencies, which we cannot really avoid (hence we use `forwardRef` decorator, when using them)